webmagic-core/src/main/java/us/codecraft/webmagic/Page.java

```diff
@@ -168,18 +168,25 @@ public void addTargetRequests(Iterable<String> requests, long priority) {
      * @param priority Priority for the URL
      */
     private void addRequestIfValid(String url, long priority) {
-        if (StringUtils.isBlank(url) || url.equals("#") || url.startsWith("javascript:")) {
-            return;
+        boolean isBlankUrl = StringUtils.isBlank(url);
+        boolean isHashSymbol = url.equals("#");
+        boolean isJavaScript = url.startsWith("javascript:");
+
+        if (isBlankUrl || isHashSymbol || isJavaScript) {
+            return; // Invalid URL, so no further processing is needed.
```

Comment:
This change does not exactly match the original version, right? cause of the shortcircut of the operator `||`.